boards/arm/96b_carbon/96b_carbon_defconfig

```diff
@@ -9,18 +9,26 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 # Enable MPU
 CONFIG_STM32_ARM_MPU_ENABLE=y
 
+CONFIG_SERIAL=y
 # enable USART1 - passthrough to FT230XQ
 CONFIG_UART_STM32_PORT_1=y
 
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
 # enable I2C1
 CONFIG_I2C=y
-CONFIG_I2C_STM32_V1=y
 
 # enable pinmux
 CONFIG_PINMUX=y
 
 # enable GPIO ports A, B, C, D
 CONFIG_GPIO=y
+CONFIG_GPIO_STM32_PORTA=y
```

Comment:
I'm not sure about GPIO activation. Can we move it under arch and "if GPIO"
Drawback is that it will activate all GPIO port by default, but might be less error prone for new comers.
And footprint optimisation can still happen in this file.

Reply:
I moved GPIO definition for ports A, B, C in common Kconfig.defconfig.series and enable the rest ports per SOC in SOC's Kconfig.defconfig.series or Kconfig.defconfig.stm32*.